core/metacling/src/TCling.cxx

```diff
@@ -6925,6 +6925,13 @@ TInterpreterValue *TCling::CreateTemporary()
    return val;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Deleting temporary value pointer
+void TCling::DeleteTemporary(TInterpreterValue* value)
+{
+   delete value;
```

Comment:
We don't need that, the caller of `CreateTemporary()` can just delete the `TInterpreterValue` object.